packages/calypso-products/src/feature-group-plan-map.ts

```diff
@@ -225,6 +226,7 @@ export const featureGroups: Partial< FeatureGroupMap > = {
 			FEATURE_NEWSLETTERS_RSS,
 			FEATURE_AD_FREE_EXPERIENCE,
 			FEATURE_WORDADS,
+			FEATURE_STATS_PAID,
```

Comment:
This is the comparison chart at the bottom of /plans/<site> and possibly other places.
<img width="1814" alt="plans-comparison" src="https://github.com/Automattic/wp-calypso/assets/140841/8fa30c2d-1c93-4807-bb96-26e6a3848f0e">